src/utils/hooks/useGetUserInfo.ts

```diff
@@ -1,26 +1,24 @@
 import { useEffect, useState } from 'react'
```

Comment:
would this method get re-fired after login ?

Reply:
I think the reason I first thought that login url and login logic should return and set the same stuff as this hook, was because this hook happen above everything else (the first hook to fire up). And wasn't sure how to re-fire it on login

Reply:
Yes, it gets called by UserArea, which can't load until log in is successful.  Is that what you meant?

(It's showing what it's supposed to in the UI and all the user data is getting loaded after login)

Reply:
hmm, I see why you wanted to store username in local storage now. I originally thought of slightly different workflow. With this way we do need to store something in local storage apart from JWT (or access JWT to figure out username is nonRegistered)

Reply:
As per discussion look good to go